src/background.js

```diff
@@ -1608,16 +1616,27 @@ function initializeGhosteryModules() {
 	]).then(() => {
 		// run scheduledTasks on init
 		scheduledTasks().then(() => {
-			// Open the Ghostery Hub on install with justInstalled query parameter set to true.
-			// We need to do this after running scheduledTasks for the first time
-			// because of an A/B test that determines which promo variant is shown in the Hub on install
-			if (globals.JUST_INSTALLED && BROWSER_INFO.name !== 'ghostery_desktop') {
-				const showAlternateHub = conf.hub_layout === 'alternate';
-				const route = showAlternateHub ? '#home' : '';
-				chrome.tabs.create({
-					url: chrome.runtime.getURL(`./app/templates/hub.html?justInstalled=true&ah=${showAlternateHub}${route}`),
-					active: true
-				});
+			if (globals.JUST_INSTALLED) {
+				(async() => {
+					await globals.BROWSER_INFO_READY;
+					// TODO comment in before 8.5.5 release (commented out to facilitate onboarding testing)
```

Comment:
@wlycdgr This makes me nervous. Do we need this? Can we trigger something from the debugger instead?

Reply:
I've removed this hack from `background` and added support for opening the Dawn hub to `ghostery.openIntroHub()` in the debugger